backend/README.md

```diff
@@ -1,5 +1,28 @@
 ## How to run database
-TODO
+Make sure you have [Docker Desktop](https://www.docker.com/get-started/) on your machine.
```

Comment:
What is this

Reply:
this was something u added when soph and I checked the logs I think

Reply:
> What is this

this was something u added when soph and I checked the logs I think